lib/ca_certs_stubs.c

```diff
@@ -0,0 +1,51 @@
+#include "caml/alloc.h"
+#include "caml/fail.h"
+#include "caml/memory.h"
+
+#ifdef _WIN32
+
+#include <windows.h>
+
+static value alloc_cons(value v_hd, value v_tl)
+{
+    CAMLparam2(v_hd, v_tl);
+
+    CAMLlocal1(v_cons);
+    v_cons = caml_alloc(2, 1);
```

Comment:
```suggestion
    v_cons = caml_alloc(2, 0);
```
cons cells have tag 0!